frontend/src/core/comments/components/AddComment.js

```diff
@@ -0,0 +1,76 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './AddComment.css';
+
+import { UserAvatar } from 'core/user'
+
+type Props = {|
+    user: string,
+    username: string,
+    imageURL: string,
+    translationId: number,
+    addComment: (string, number) => void,
+|};
+
+
+export default function AddComments(props: Props) {
+    const {
+        user,
+        username,
+        imageURL,
+        translationId,
+        addComment,
+    } = props;
+
+    let commentInput: any = React.useRef();
+
+    if (!user) {
+        return null;
+    }
+
+    const onEnterSubmit = (event: SyntheticKeyboardEvent<HTMLTextAreaElement>) => {
+        if (event.keyCode === 13 && event.shiftKey === false) {
+            event.preventDefault();
+            submitComment(event);
+        }
+    }
+
+    const submitComment = (event: SyntheticKeyboardEvent<>) => {
+        event.preventDefault();
+        const comment = commentInput.current.value;
+
+        if (!comment) {
+            return null;
+        }
+
+        addComment(comment, translationId);
+        commentInput.current.value = '';
+    };
+
+    return <div className='comment add-comment'>
+        <UserAvatar
+            user={ user }
+            username={ username }
+            title=''
+            imageUrl={ imageURL }
+        />
+        <form className='container'>
+            <Localized
+                id='comments-AddComment--input'
+                attrs={{ placeholder: true }}
+            >
+                <textarea
+                    id='comment-input'
+                    name='comment'
+                    dir='auto'
+                    placeholder={ `Write a comment ${'\u2026'}` }
```

Comment:
No space before the ellipsis, and I believe you can simply use the actual character, no need to use the Unicode representation: `…` (you can just copy and paste it ;) ).

Reply:
Note that, whenever you change localized content, what actually matters is what is in the `en-US` FTL file, *not* what is in the code. That's there only to help developers understand what the content is without having to go look into the `.ftl` files, but that content will never be shown to users. We should aim to have content in code always in sync with content in the `en-US` reference FTL files, even if that's a bit painful to do.